app/src/main/java/org/wikipedia/util/FileUtil.java

```diff
@@ -97,5 +98,30 @@ public static String bytesToUserVisibleUnit(Context context, long bytes) {
         return context.getString(R.string.size_mb, bytesToMB(bytes));
     }
 
+    static void writeToFileInDirectory(String data, String directory, String filename) {
+        File file = new File(directory, filename);
+
+        FileOutputStream outputStream = null;
+        try {
+            file.createNewFile();
+            outputStream = new FileOutputStream(file, true);
```

Comment:
The `true` parameter causes the contents to be appended to the end of the file, instead of re-creating the file with the new data.  This can simply be `new FileOutputStream(file, true)`, which defaults to `false`.

Reply:
Done.